### PR TITLE
Update the createacreaturecard Command: now with embed and editmenu

### PR DESCRIPTION
**Pull request by TheMBeat**

### CODE REVIEW

File: mindbot.py [hunk 1]
```diff
@@ -35,12 +36,201 @@ async def checkUserId(interaction: discord.Interaction, usertag: str):
...
+	uid_from_setInput = ui.TextInput(label='Cardnumber', placeholder="1/24", required=True)
+	
+	async def on_submit(self, interaction: discord.Interaction):
+
+		await interaction.response.defer(ephemeral=True, thinking=True)
+		# Update the Mindbug-Card
+		name, power, capabilities, effect, quote, cardnumber, setname, language, filename, override = getCardDataFromEmbed(interaction)
+
+		myEmbed = discord.Embed(color = discord.Color.random())
+		myEmbed.add_field(name="Name", value=name, inline=True) # Index 0
```
Comment (rlapostolle) [inline mindbot.py:136]: in the field description, would be good to have the allowed values (like true/false for override or "en" for default language)

File: mindbot.py [hunk 2]
```diff
@@ -189,24 +367,18 @@ async def createcreaturecard(interaction: discord.InteractionMessage, artwork :
...
-				
-				await interaction.followup.send(file=discord.File(fp=image_binary, filename=artwork.filename))
+			myEmbed = discord.Embed(color = discord.Color.random())
+			myEmbed.add_field(name="Name", value="Sirus Snape", inline=True) # Index 0
+			myEmbed.add_field(name="Power", value="X", inline=False) # Index 1
+			myEmbed.add_field(name="Capabilities", value="?", inline=False) # Index 2
+			myEmbed.add_field(name="Effect", value= "?"  , inline=False) # Index 3
+			myEmbed.add_field(name="Quote", value= "?" , inline=False) # Index 4
+			myEmbed.add_field(name="Cardnumber in Set", value="1/1", inline=False) # Index 5
+			myEmbed.add_field(name="Name from Set", value="Community Raid", inline=False) # Index 6
```
Comment (rlapostolle) [inline mindbot.py]: better to have "default" as default value instead of Community Raid

File: mindbot.py [hunk 3]
```diff
@@ -35,12 +36,201 @@ async def checkUserId(interaction: discord.Interaction, usertag: str):
...
+			myEmbed.add_field(name="Capabilities", value=capabilities, inline=False) # Index 2
+			myEmbed.add_field(name="Effect", value=effect, inline=False) # Index 3
+			myEmbed.add_field(name="Quote", value=quote, inline=False) # Index 4
+			myEmbed.add_field(name="Cardnumber in Set", value=cardnumber, inline=False) # Index 5
+			myEmbed.add_field(name="Setname", value=setname, inline=False) # Index 6
+			myEmbed.add_field(name="Language", value=language, inline=False) # Index 7
+			myEmbed.add_field(name="Filename", value=f"{filename}", inline=False) # Index 8
+			myEmbed.add_field(name="Override", value=f"{override}", inline=False) # Index 9
+		
+			await interaction.followup.send(embed = myEmbed, view = EditMenu(), file=discord.File(fp=image_binary, filename=filename), ephemeral=True)
```
Comment (rlapostolle) [inline mindbot.py:197]: it would be cool that the old post is removed when the updated menu show up

Comment (TheMBeat) [reply mindbot.py:197]: I have not found a method yet.

**Review by rlapostolle — APPROVED** [review]

**Pull request merged**